pelican_gist/plugin.py

```diff
@@ -108,6 +108,10 @@ def replace_gist_tags(generator):
     should_cache = generator.context.get('GIST_CACHE_ENABLED')
     cache_location = generator.context.get('GIST_CACHE_LOCATION')
     pygments_style = generator.context.get('GIST_PYGMENTS_STYLE')
+    
+    # added by SgtDaJim on 20170809
+    # To solve the problem that it alerts "local variable 'body' referenced before assignment" when compiling blogs.
```

Comment:
OK, I'm sorry. I just get used to mark every changes in the code, to avoid forgetting what I did..